packages/terra-application-header-layout/examples/index-examples/HeaderWireframe.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
-import ApplicationHeaderLayout from 'terra-application-header-layout';
-import ResponsiveElement from 'terra-responsive-element';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import ResponsiveElement from 'terra-responsive-element/lib/ResponsiveElement';
+import ApplicationHeaderLayout from '../../lib/ApplicationHeaderLayout';
```

Comment:
I want to start some discussion about pulling this from `'../../src/ApplicationHeaderLayout'`. 

This will enable live reload of the component src but there are some things to think through around if this means we are viewing ES6 code or ES5 code in the webpack dev server.